com.github.xournalpp.xournalpp.yaml

```diff
@@ -14,6 +13,7 @@ finish-args:
   # Wayland access
   - "--socket=wayland"
   # Needs to save files locally
+  # Why are portals not used? Does the app re-implement the open dialogue?
```

Comment:
This should be changed to a description why we require this. E.g. Xournal++ supports references to external files within their fileformat, hence global file access is required.